### ISSUE TITLE
[Feature Request] Improve the Readability of Search Pad

### ISSUE THREAD

**Issue by imchell** [nextui-org/nextui#576]
### Is your feature request related to a problem? Please describe.

Currently search pad lacks color contrast, making it hard to recognize captions.

### Describe the solution you'd like

Change the color of description texts (both default state and hover state).

### Screenshots or Videos

![Search Pad](https://us1.myximage.com/2022/06/26/77455c16d4dc3f9a5e2a36b992af8554.png)

**Comment by mattbudde** [comment]
i'll have a PR up to address this shortly.
